projects/step-frontend/src/lib/modules/timeseries/modules/legacy/components/timeseries-table/timeseries-table.component.ts

```diff
@@ -382,7 +382,7 @@ export class TimeseriesTableComponent implements OnInit, OnDestroy {
 
   ngOnDestroy(): void {
     this.tableData$.complete();
-    this.terminator$.next();
+    this.terminator$.next(undefined);
```

Comment:
```suggestion
    this.terminator$.next();
```

Reply:
@dvladir even though the file compiles, my IDE started to show error on these things (e.g 1 parameter is required). very weird..

Reply:
@lucianbaciu mine doesn't show these errors and it actually doesn't make sense. Maybe investigate why your IDE does this / if there are updates that might solve it.